channel2/impl.go

```diff
@@ -201,10 +201,14 @@ func (channel *channelImpl) SendWithPriority(m *Message, p Priority) (err error)
 		}
 	}()
 	channel.stampSequence(m)
-	channel.outQueue <- &priorityMessage{m: m, p: p}
+	channel.enqueue(m, p)
```

Comment:
Not interested in holding up this PR for it, but I don't personally think this reads any clearer than what was already there, and it introduces the overhead of a function call.

Reply:
I was experimenting with a separate queue for higher priority messages, to see if that made any difference in getting control messages through in a timely fashion. It didn't seem to make any difference, but this was a remnant from that experiment. I went ahead and removed it.